tests/balance.rs

```diff
@@ -43,9 +43,194 @@ fn verify_get_balance() {
         let res = MoveModule::execute(
             RuntimeOrigin::signed(addr_native.clone()),
             transaction_bc,
-            MAX_GAS_AMOUNT,
+            100,
```

Comment:
Okay - this doesn't look nice - let's rather use a hardcoded value for the `cheque_amount` in the lib.rs and decouple it from the `GasAmount` parameter...

Reply:
Restored it to the original state - the test now passes, because I disabled the entry-check for sufficient balance on the user's account - because this shall be coupled to the additional function paramter of the `execute` extrinsic...